package.json

```diff
@@ -12,12 +12,18 @@
     "clients:js:lint": "zx ./scripts/client/lint-js.mjs",
     "clients:js:publish": "zx ./scripts/client/publish-js.mjs",
     "clients:js:test": "zx ./scripts/client/test-js.mjs",
+    "clients:rust:check": "pnpm clients:rust:format && pnpm clients:rust:lint && pnpm clients:rust:features && pnpm clients:rust:rustdoc",
```

Comment:
I'm not against an aggregate `pnpm` script that runs format, lint, and test all in one go.